src/GraphQL/Validation/ValidationContext.cs

```diff
@@ -148,22 +186,42 @@ public IEnumerable<FragmentDefinition> GetRecursivelyReferencedFragments(Operati
             return fragments;
         }
 
+        /// <summary>
+        /// Returns a string representation of the specified node.
+        /// </summary>
         public string Print(INode node)
         {
             return AstPrinter.Print(node);
         }
 
+        /// <summary>
+        /// Returns the name of the specified graph type.
+        /// </summary>
         public string Print(IGraphType type)
         {
             return SchemaPrinter.ResolveName(type);
         }
     }
 
+    /// <summary>
+    /// Represents a variable reference node and the graph type it is referenced to be used for.
```

Comment:
I'm not a native speaker and this comment seems hard to understand.

Reply:
Yeah I know...... any ideas? Sometimes I just draw a blank about what to write.

Reply:
OK... let's keep it as is.